static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -23,6 +24,35 @@ declare class YoutubePlayerEvent {
 }
 
 const players = {};
+const iframes = [];
+
+document.addEventListener('focusout', () => {
+    iframes.forEach(iframe => {
+        fastdom
+            .read(() => {
+                if (document.activeElement === iframe) {
+                    return $('.vjs-big-play-button', iframe.parentElement);
+                }
+            })
+            .then(($playButton: ?bonzo) => {
+                fastdom.write(() => {
+                    if ($playButton) {
+                        $playButton.addClass('youtube-play-btn-focussed');
+                    }
+                });
+            });
+    });
+});
+
+document.addEventListener('focusin', () => {
+    fastdom.read(() => $('.vjs-big-play-button')).then(($playButton: bonzo) => {
```

Comment:
Should the type of `$playButton` be `?bonzo`?

Reply:
Yeah I reckon! 👍